src/app/public/modules/text-editor/services/text-merge-field.service.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  Injectable
+} from '@angular/core';
+
+import {
+  SkyAppWindowRef
+} from '@skyux/core';
+
+/**
+ * @internal
+ */
+@Injectable()
+export class SkyTextMergeFieldService {
```

Comment:
Following convention I think this would normally be `SkyTextMergeFieldAdapterService` since it touches the DOM. Thoughts?

Reply:
It's a good thought. I think this is a good candidate to be wrapped up completely inside the text editor adapter. Moved and renamed `getMergeFieldDataURI` to be more self-explanatory.